roles/disk/README.rst

```diff
@@ -0,0 +1,9 @@
+Disk
```

Comment:
I tried but there is an ordering issue. the current lvm role has to run before the docker and (optional) mesos roles but this has to run after both (and any other future role that might want a slice of the disk) to allocate the remainder of the space. If we pull all of the lvm stuff out of the docker and mesos roles, we might be able to get there but then the lvm role will have to know about some docker and mesos configs which I thought might be confusing with our current var setup. If you have another idea how to do it, I'd prefer to consolidate it too. @avnik is there anything we can do with mantl-storage-setup to help with this?